README.md

```diff
@@ -59,11 +65,12 @@ example-organization
 
 #### Logs
 
-Among the four projects created under the common folder, two projects (prj-c-logging, prj-c-org-billing-logs) are used for logging. The first one for organization wide audit logs and another for billing logs.
+Among the four projects created under the common folder, two projects (`prj-c-logging`, `prj-c-org-billing-logs`) are used for logging.
```

Comment:
They are six projects now:
- prj-c-billing-logs
- prj-c-dns-hub
- prj-c-interconnect
- prj-c-logging
- prj-c-scc
- prj-c-secrets


and `prj-c-org-billing-logs` is `prj-c-billing-logs`